ophyd/areadetector/trigger_mixins.py

```diff
@@ -71,41 +80,68 @@ class MultiTrigger(TriggerBase):
 
     This can be used to give more control to the detector. One call to
     'trigger' can be interpreted by the detector as a call to take several
-    acquisitions with, for example, different gain settings.
+    acquisitions with, for example, different gain settings or shutter
+    positions.
+
+    The are two levels of nesting here:
+    - cycling through different actions on successive calls to `trigger`
+    - within each trigger, executing a list of acquisitions with different
+      settings
 
-    There is no specific logic implemented here, but it provides a pattern
-    that can be easily modified. See in particular the method `_acquire` and
-    the attribute `_num_acq_remaining`.
+    See the example below, which takes and 3 and 1 acquisitions in
+    alternation.
 
     Example
     -------
     >>> class MyDetector(SimDetector, MultiTrigger):
     ...     pass
-    >>> det = MyDetector(acq_cycle={'image_gain': [1, 2, 8]})
+    # On the first trigger, close the shutter and acquire three images
+    # with different gain settings on the detector. Then open the shutter
+    # and take a light frame.
+    # On the next trigger, just take a light frame.
+    >>> trigger_cycle=[[('gain1', {'shutter': 'close', 'image_gain': 1}),
```

Comment:
I'm laughing because I was just criticizing @tacaswell for leaving docstrings out of his PR. I should have known better than to include docstrings. :-P